SWLOR.Game.Server/Service/Spawn.cs

```diff
@@ -318,12 +318,14 @@ public static void QueueRespawn()
             uint creature = OBJECT_SELF;
             var spawnId = GetLocalString(creature, "SPAWN_ID");
             if (string.IsNullOrWhiteSpace(spawnId)) return;
+            if (GetLocalInt(creature, "RESPAWN_QUEUED") == 1) return;
```

Comment:
What does this correct?

Reply:
Sometimes when I scavenged a ScavengePoint it would respawn twice.  Presumably because somehow the destroy event was firing multiple times.  It's general best practice to put this sort of logic around events that are only meant to happen once - stuff firing multiple times seems to be pretty endemic in NWN.